design-system/materials/custom-properties_recolouring.css

```
@@ -15,44 +15,45 @@
   --color-primary-85: hsl(244, 100%, 84%);
   --color-primary-90: hsl(243, 100%, 93%);
   --color-primary-95: hsl(244, 100%, 97%);
+  --color-primary-98: hsl(244, 100%, 99%);
   --color-accent: #213c45;
-  --color-accent-10: hsl(195, 35.2941176471%, 10%);
-  --color-accent-20: hsl(195, 35.2941176471%, 20%);
-  --color-accent-30: hsl(195, 35.2941176471%, 30%);
-  --color-accent-40: hsl(195, 35.2941176471%, 40%);
-  --color-accent-50: hsl(195, 35%, 50%);
-  --color-accent-60: hsl(195, 35.2941176471%, 60%);
-  --color-accent-85: hsl(195, 35%, 85%);
-  --color-accent-90: hsl(195, 35.2941176471%, 90%);
-  --color-accent-95: hsl(195, 35.2941176471%, 95%);
-  --color-accent-98: hsl(195, 35.2941176471%, 98%);
-  --color-brown-10: hsl(35, 90%, 10%);
-  --color-brown-20: hsl(35, 50%, 20%);
-  --color-brown-35: hsl(35, 25%, 35%);
-  --color-brown-50: hsl(35, 30%, 50%);
-  --color-brown-70: hsl(35, 40%, 70%);
-  --color-brown-85: hsl(35, 60%, 85%);
-  --color-brown-90: hsl(35, 90%, 90%);
-  --color-brown-95: hsl(35, 80%, 95%);
-  --color-brown-98: hsl(35, 90%, 98%);
-  --color-purple-10: hsl(248, 90%, 10%);
+  --color-accent-10: hsl(195, 100%, 10%);
+  --color-accent-20: hsl(195, 80%, 20%);
+  --color-accent-30: hsl(195, 70%, 30%);
+  --color-accent-40: hsl(195, 70%, 40%);
+  --color-accent-50: hsl(195, 70%, 50%);
+  --color-accent-60: hsl(195, 70%, 60%);
+  --color-accent-85: hsl(195, 85%, 85%);
+  --color-accent-90: hsl(200, 90%, 90%);
```

The `hue` value from the figma table here is 195. Perhaps there is a reason why you use 200? 
```suggestion
  --color-accent-90: hsl(195, 90%, 90%);
```

---

Thanks @ddouglasz, great catches 🙌 Those values have changed over time, missed those few updates. Updated in b432d630f2602f7ace57dad982859b8652507926